src/main/drivers/pwm_output.c

```diff
@@ -352,6 +353,28 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         }
     }
 }
+
+void pwmWriteDshotCommandAll(uint8_t motorCount, uint8_t command)
+{
+    if (command <= DSHOT_MAX_COMMAND) {
```

Comment:
I was getting missed beeps on the esc, thought this could have been the issue.. but I don't think it is now.